api/v1alpha1/serviceintentions_webhook.go

```diff
@@ -41,7 +43,10 @@ func (v *ServiceIntentionsWebhook) Handle(ctx context.Context, req admission.Req
 		return admission.Errored(http.StatusBadRequest, err)
 	}
 
-	svcIntentions.Default()
+	defaultingPatches, err := v.defaultingPatches(err, &svcIntentions)
+	if err != nil {
+		return admission.Errored(http.StatusInternalServerError, err)
```

Comment:
Not sure if this should be 500 or 400. If there's an error, it will happen every time but the error could be due to the input or our internal defaulting logic.

Reply:
You're right. If malformed JSON makes it here, it's probably because the API server messed up in some weird way. I'm OK with a 500 here, given i'd expect the API server to return a 400